src/pages/AddMembersPage/AddMembersPage.tsx

```diff
@@ -117,6 +118,7 @@ const AddMembersPage = () => {
       QUERY_KEY_GET_MEMBERS_PAGE_DATA,
       circleId,
     ]);
+    queryClient.invalidateQueries(QUERY_KEY_GET_ORG_MEMBERS_DATA);
```

Comment:
@sshmm this doesn't have an await... intentional or a mistake?

Reply:
it's intentional. I believe that is unnecessary to await as it mark the query as stale.
when the user navigate to org member page it will refetch the quey